frameworklauncher/src/main/java/com/microsoft/frameworklauncher/common/utils/CommonUtils.java

```diff
@@ -140,4 +140,10 @@ public static String readFile(String filePath) throws Exception {
     System.arraycopy(arrayTail, 0, result, arrayHead.length, arrayTail.length);
     return result;
   }
+
+  public static int bytesToShort(byte[] bytes) {
+    int low = (bytes[0] & 0xFF);
```

Comment:
>( [](start = 14, length = 1)

Parentheses seemed not necessary here.

Reply:
Maybe, but here for safety :)

---
In reply to: [170504425](https://github.com/Microsoft/pai/pull/338#discussion_r170504425) [](ancestors = 170504425)

Reply:
I'm sure it not useful as this is a rather simple statement :)

---
In reply to: [170504760](https://github.com/Microsoft/pai/pull/338#discussion_r170504760) [](ancestors = 170504760,170504425)

Reply:
Yes, verified. Will remove Parentheses 

---
In reply to: [170507379](https://github.com/Microsoft/pai/pull/338#discussion_r170507379) [](ancestors = 170507379,170504760,170504425)